api/src/index.js

```diff
@@ -2,30 +2,54 @@ import "dotenv/config";
 import express from "express";
 import cors from "cors";
 import bodyParser from "body-parser";
-import knex from "./database_client.js";
-import nestedRouter from "./routers/nested.js";
+import allMealsRouter from "./routers/all-meals.js";
+import firstMealRouter from "./routers/first-meal.js";
+import lastMealRouter from "./routers/last-meal.js";
+import futureMealsRouter from "./routers/future-meals.js";
+import pastMealsRouter from "./routers/past-meals.js";
+
 
 const app = express();
 app.use(cors());
 app.use(bodyParser.json());
 
 const apiRouter = express.Router();
 
-// You can delete this route once you add your own routes
+
 apiRouter.get("/", async (req, res) => {
-  const SHOW_TABLES_QUERY =
-    process.env.DB_CLIENT === "pg"
-      ? "SELECT * FROM pg_catalog.pg_tables;"
-      : "SHOW TABLES;";
-  const tables = await knex.raw(SHOW_TABLES_QUERY);
-  res.json({ tables });
+  // res.json({ message : "Welcome to meal sharing API" ,
+  //   description: "This API provides routes to view meal information.",
+  //   routes: {
+  //     "/api/future-meals": "Meals scheduled for the future.",
+  //     "/api/past-meals": "Meals that have already occurred.",
+  //     "/api/all-meals": " Get All meals that are sorted by ID.",
+  //     "/api/first-meal": "Get the meal with the smallest ID.",
+  //     "/api/last-meal": "Get the meal with the largest ID.",
+  //   }
+  // });
+  res.send(`<h1>Welcome to meal sharing API</h1>
+            <p>This API provides routes to view meal information</p>
+            <ul>
+            <li><strong>/api/future-meals":</strong> "Meals scheduled for the future.</li>
+           <li><strong>/api/past-meals":</strong> "Meals that have already occurred.</li>
+          <li><strong>/api/all-meals": "</strong> Get All meals that are sorted by ID.</li>
+          <li><strong>/api/first-meal":</strong> "Get the meal with the smallest ID.</li>
+          <li><strong>/api/last-meal":</strong> "Get the meal with the largest ID.</li>
+          </ul>
+      `);
```

Comment:
Checkout here in my inclass example of the CRUD app:
https://github.com/HackYourFuture-CPH/node.js/tree/team29/week1/inclass/inclass/example-crud-backend

you will need another middleware and some more lines of code and installed packages:
```
const __filename = fileURLToPath(import.meta.url);
const __dirname = dirname(__filename);
app.use(express.static(path.join(__dirname, 'public')));
```

and this route:
```
// Add this route before other routes
app.get('/', (req, res) => {
    console.log("Request for index.html received");
    res.sendFile(path.join(__dirname, 'public', 'index.html'));
});
```

but then you can serve, through this endpoints, the index.html file from the public folder:
https://github.com/HackYourFuture-CPH/node.js/tree/team29/week1/inclass/inclass/example-crud-backend/public

this is cleaner than writing the HTML here into the JS file :)